### PR TITLE
Added translation to Polish

### PR DESCRIPTION
**Pull request by szafranski**

### CODE REVIEW

File: doc/pl-Software Setup.md [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+Automatyczne numerowanie sprzętowe, również jest możliwe. Ustaw NODE_NUMBER jako 0, następnie połącz następujące piny z masą:
+
+node #1: ground pin D5<br/>
+node #2: ground pin D6<br/>
+node #3: ground pin D7<br/>
+node #4: ground pin D8<br/>
+node #5: ground pin D5 and pin D4<br/>
+node #6: ground pin D6 and pin D4<br/>
+node #7: ground pin D7 and pin D4<br/>
+node #8: ground pin D8 and pin D4
```
Comment (HazardCreative) [inline doc/pl-Software Setup.md]: Did you want to translate these?

File: doc/pl-Hardware Setup.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            },
+            "i2c:0x40": {
+                    "name": "Battery",
+                    "max_current": 0.1
+            },
+            "i2c:0x41": {
+                    "name": "Pi",
+                    "max_current": 2
+            }
+    },
```
Comment (HazardCreative) [inline doc/pl-Hardware Setup.md]: You may translate the labels (Climate, Battery, Pi) if you would like.

**Comment by szafranski** [comment]
I would like to. I cliked on resolve button by mistake. What can I do now to translatw those two?

**Comment by HazardCreative** [comment]
Simply update the files on your own branch that you submitted the PR from.

**Comment by szafranski** [comment]
Done

**Comment by HazardCreative** [comment]
Will you be adding any new strings to language.json as well? See https://github.com/RotorHazard/RotorHazard/commit/4c4951985f03b9ddbe1355cf628dfed5f5dc8d5b

**Comment by szafranski** [comment]
For now, no. I will consider it after testing server software someday,
switched Polish.

W dniu pon., 2.03.2020 o 19:19 Hazard Creative <notifications@github.com>
napisał(a):

**Comment by szafranski** [comment]
Great, thanks.

W dniu pon., 2.03.2020 o 19:26 Hazard Creative <notifications@github.com>
napisał(a):

**Pull request merged**